ccdproc/image_collection.py

```diff
@@ -689,16 +689,19 @@ def _generator(self, return_type,
 
         for full_path in self._paths():
             no_scale = do_not_scale_image_data
-            hdulist = fits.open(full_path,
-                                do_not_scale_image_data=no_scale)
+            if return_type == 'ccd':
+                ccddata = fits_ccddata_reader(full_path, **ccd_kwargs)
```

Comment:
Oh, totally overlooked this.

Currently it's not possible to use `do_not_scale_image_data` because it's ["unsupported"](https://github.com/astropy/ccdproc/commit/293e31ed7706bc507d02bc03692442ea7d6eb466#diff-b3346e64cf4de448f0fab8c089b63c0dR189)